book/src/operations.md

```diff
@@ -1 +1,29 @@
+
+More precisely, an operation is a code (or, in the frontend, string identifier) followed by 0 or more arguments. These arguments may consist of up to three statements, up to one key-value pair and up to one Merkle proof.
+
+The following table summarises the natively-supported operations:
+
+| Code | Identifier         | Args                | Condition                                                                                                             | Output                                                         |
+|------|--------------------|---------------------|-----------------------------------------------------------------------------------------------------------------------|----------------------------------------------------------------|
+| 0    | `None`             |                     |                                                                                                                       | `None`                                                         |
+| 1    | `NewEntry`         | `(key, value)`      |                                                                                                                       | `ValueOf(ak, value)`, where `ak` has key `key` and origin ID 1 |
+| 2    | `EntryEq`          | `s1`, `s2`          | `s1 = ValueOf(ak1, value1)`, `s2 = ValueOf(ak2, value2)`, `value1 = value2`                                           | `Equal(ak1, ak2)`                                              |
+| 3    | `SymmetricEq`      | `s`                 | `s = Equal(ak1, ak2)`                                                                                                 | `Equal(ak2, ak1)`                                              |
+| 4    | `TransitiveEq`     | `s1`, `s2`          | `s1 = Equal(ak1, ak2)`, `s2 = Equal(ak3, ak4)`, `ak2 = ak3`                                                           | `Equal(ak1, ak4)`                                              |
+| 5    | `EntryNEq`         | `s1`, `s2`          | `s1 = ValueOf(ak1, value1)`, `s2 = ValueOf(ak2, value2)`, `value1 != value2`                                          | `NotEqual(ak1, ak2)`                                           |
+| 6    | `SymmetricNEq`     | `s`                 | `s = NotEqual(ak1, ak2)`                                                                                              | `NotEqual(ak2, ak1)`                                           |
+| 7    | `GtToNonequality`  | `s`                 | `s = Gt(ak1, ak2)`                                                                                                    | `NotEqual(ak1, ak2)`                                           |
```

Comment:
I observe 3 different ways to write the equality/non-equality part of a statement/operation:
- Eq, Equal
- NEq, NotEqual, Nonequality
Can we unify them and pick 1 for equality and 1 for nonequality and use it everywhere?